.github/workflows/installcheck.yml

```diff
@@ -0,0 +1,40 @@
+name: PG12 Regression
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get latest commit id of PostgreSQL 12
+        run: |
+          echo "PG_COMMIT_HASH=$(git ls-remote git://git.postgresql.org/git/postgresql.git refs/heads/REL_12_STABLE | awk '{print $1}')" >> $GITHUB_ENV
+
+      - name: Cache PostgreSQL 12
+        uses: actions/cache@v2
+        id: pg12cache
+        with:
+          path: ~/pg12
+          key: ${{ runner.os }}-v1-pg12-${{ env.PG_COMMIT_HASH }}
+
+      - name: Install PostgreSQL 12
+        if: steps.pg12cache.outputs.cache-hit != 'true'
+        run: |
+          git clone --depth 1 --branch REL_12_STABLE git://git.postgresql.org/git/postgresql.git ~/pg12source
+          cd ~/pg12source
+          ./configure --prefix=$HOME/pg12 CFLAGS="-std=gnu99 -ggdb -O0" --enable-cassert
+          make install -j$(nproc) > /dev/null
+
+      - uses: actions/checkout@v2
+      - name: Regression
+        run: |
+          export PG_CONFIG=$HOME/pg12/bin/pg_config 
+          make -j$(nproc)
+          make install
+          make installcheck
```

Comment:
Why is this?

Reply:
Actually, if we follow all PostgreSQL coding rules, it will be easy to run pgident and merge.  All of the extensions are doing that. But it's my experience and opinion.

Reply:
But, this means all of PGs coding rules, correct? Including their indentations and use of tabs? We left all that behind at the start of the project.

Reply:
Don't worry, we will fix them one by one afterwards, for indentation, I will run pgident that fixes space and indentation issues.

Reply:
What indentation issues? Our coding standard is to use 4 spaces (not a tab) for indentation. Please do not automatically run anything that will reformat files in the AGE repo. Changes like this need to be reviewed by the group before any action is taken.

Reply:
I agree, @jrgemignani, that is what we already decided, and I don't want to run any tool without a final agreement. My point is our code is based on PostgreSQL, and for PostgreSQL,  even we have copied most of the code from PostgreSQL code. We should make some progress to match PostgreSQL coding guidelines. Not now, but start thinking over that. It will ease our job of the future merging of code, and we can also use PostgreSQL indentation tools in future.